env0/resource_environment.go

```diff
@@ -628,8 +628,25 @@ func getTTl(date string) client.TTL {
 	}
 }
 
+func getConfigurationVariables(apiClient client.ApiClientInterface, environmentId string) ([]client.ConfigurationVariable, error) {
+	variables, err := apiClient.ConfigurationVariablesByScope(client.ScopeEnvironment, environmentId)
+	if err != nil {
+		return nil, err
+	}
+
+	// Filter out template/blueprint variables.
+	filteredVariables := []client.ConfigurationVariable{}
+	for _, variable := range variables {
+		if variable.Scope != client.ScopeTemplate {
```

Comment:
global is already filtered out in the API call.

Reply:
Blueprint as the same as global (kinda), if we fetch by Environment or Deployment scope, then we might get back Blueprint scope variables.

I think this filter logic should be in the API call itself

Reply:
@RLRabinowitz - modified to the API call itself.